tests/test_miscellaneous.py

```diff
@@ -18,4 +17,29 @@ def test_properties_getters(self):
     @not_raises((AttributeError, ValueError))
     def test_properties_setters(self):
         self.subject.metric_collector_name = "test"
-        self.subject.languages = ["test"]
\ No newline at end of file
+        self.subject.languages = ["test"]
+
+    def test_asdict(self):
+        dict_ = self.subject._asdict()
+        assert_equal(dict_['languages'], ["RUBY"])
+        assert_equal(dict_['metric_collector_name'], "MetricFu")
+
+
+class TestCompoudMetric(TestCase):
+    def setUp(self):
+        self.subject = CompoundMetricFactory.build()
+
+    def test_properties_getters(self):
+        assert_true(hasattr(self.subject, 'script'))
+
+    @not_raises((AttributeError, ValueError))
+    def test_properties_setters(self):
+        self.subject.script = "test"
+
+    @raises(ValueError)
+    def test_properties_setters(self):
+        self.subject.script = None
+
+    def test_asdict(self):
+        dict_ = self.subject._asdict()
+        assert_equal(dict_['script'], "return 0;")
```

Comment:
Instead of asserting with `"return 0;"` what do you think about assert `self.subject.script`? This way, if the factory change the test is not broken.